rollup.config.js

```diff
@@ -12,31 +12,37 @@ const ENV_ALIASES = {
 }
 const ENTRY_FILES = ['factory', 'index', 'createElement']
 
-const plugins = () =>
-  [
+const plugins = format => {
+  return [
     replace({
       'process.env.NODE_ENV': JSON.stringify(ENV),
     }),
-    babel({
-      exclude: 'node_modules/**',
-    }),
+    // Don't transpile ESM
```

Comment:
Could u elaborate on the reasoning here? Like - why main (CJS) is transpiled, module (ESM) is not? Usually they just differ in the module system which is used, but other than that they are the same.

Reply:
thanks for taking a look! Tbh I’d transpile everything except import/export but recently folks argued that it is probably not cool https://mobile.twitter.com/rob_dodson/status/1172302699204755458 
do tou normally transpile both?

Reply:
> do tou normally transpile both?

Yes - but just because we are kinda stuck in a status quo and most of the libraries are set up like this.

You could push things forward by not transpiling (people need to start doing it), but IMHO the same should be done for both main & module entries. So basically transpile or not - your choice, just be consistent for both entries.

Reply:
No one can force you to transpile down to es5, even though many people would expect it. Just document what is expected from consumers wanting to use your package in older environments (so basically advise transpiling it on their own).

Reply:
Cool thank your inputs, I really appreciate that!